restrict_edition_to_superuser/readme/USAGE.rst

```diff
@@ -0,0 +1 @@
+Set the restriction boolean to True on your record then enjoy peace
```

Comment:
I dont see how ? There is no code to add this field on the form view. Right ?

Reply:
That's right, this module has its own opinion on how it should be used, but the principle is that some records, you just don't want anyone touching them

Reply:
@legalsylvain the main aim it to flag record created with data (in your module) that should be only edited using data.